class JoinInException(Exception):
    pass


class PaymentException(JoinInException):
    pass
